frontend/html/index.html

```diff
@@ -4,12 +4,25 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="styles.css">
+    <title>Senzr</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Montserrat:wght@300&display=swap" rel="stylesheet">
```

Comment:
Is 300 the only font-weight we want? I guess that works for now but that mean we won't be able to use the bold version of this font until we add it here.

Reply:
Also great job that you did not include all weights directly. Adding a lot of variants here really lowers performance a lot. 👍

Reply:
True. I chose a font-weight of 300 to uniform the text and avoid different styles. However, this can be changed of course. I thought it would be according to some kind of style guide.